www/caddy/src/opnsense/mvc/app/models/OPNsense/Caddy/Caddy.xml

```diff
@@ -135,8 +135,7 @@
                 </enabled>
                 <FromDomain type="HostnameField">
                     <Required>Y</Required>
-                    <ValidationMessage>Please enter a valid 'from' domain or IP address.</ValidationMessage>
-                    <IpAllowed>Y</IpAllowed>
```

Comment:
I think this must be set to 'N' explicitly, as it's allowed by default: https://github.com/opnsense/core/blob/master/src/opnsense/mvc/app/models/OPNsense/Base/FieldTypes/HostnameField.php#L56

Reply:
Oh thanks, I missed that.